webpack.config.js

```diff
@@ -179,31 +172,20 @@ const config = (module.exports = {
       outputPath: __dirname + "/resources/frontend_client/app/dist",
     }),
     new webpack.DefinePlugin({
-      "process.env": { NODE_ENV: JSON.stringify(NODE_ENV) },
+      process: { env: { NODE_ENV: JSON.stringify(NODE_ENV) } },
```

Comment:
I'm getting a warning about conflicting `NODE_ENV` values that is resolved by removing this line. I think webpack sets this on its own so no reason to add it via `DefinePlugin`

Reply:
Found this

https://webpack.js.org/plugins/define-plugin/

> Warning
> When defining values for process prefer `'process.env.NODE_ENV': JSON.stringify('production')` over `process: { env: { NODE_ENV: JSON.stringify('production') } }`. Using the latter will overwrite the process object which can break compatibility with some modules that expect other values on the process object to be defined.

@daltojohnso so you think we can just remove this all together?

Reply:
oooor, let's simply use what they have on their webpage as an example?

`'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),`

agreed @ariya @daltojohnso ?

Reply:
interestingly, I still get the warning when using `'process.env.NODE_ENV'`. trying to figure out what's causing the conflict

Reply:
Sanity check for what what @daltojohnso said (that we don't have to add it manually). Created a file `foo.js` - has nothing to do with the webpack config.

```javascript
// foo.js

process.env.NODE_ENV = "development";

const bar = JSON.stringify(process.env.NODE_ENV);

console.log(process.env.NODE_ENV); // development
console.log(process.env.NODE_ENV === "development"); // true
console.log(bar); // "development"
```

Reply:
> interestingly, I still get the warning when using `'process.env.NODE_ENV'`. trying to figure out what's causing the conflict

Me too. Removing the line completely removed the error. This experiment with the `foo.js` shows that we should be good even without stringifying `NODE_ENV` value for the sake of strict equality checks.